JSQMessagesViewController/Model/JSQMessageData.h

```diff
@@ -81,8 +81,6 @@
  */
 - (NSUInteger)messageHash;
 
-@optional
```

Comment:
oops. this shouldn't have been removed. 

--------

my previous comment was for removing `@optional` from `JSQMessageMediaData.h`. however, let's actually leave `JSQMessageMediaData.h` how you have it, since copying `JSQVideoMediaItem` is not yet implemented